bin/launch_check_model.py

```diff
@@ -20,6 +20,7 @@ def get_args():
     parser.add_argument("-m", "--model", type=str, required=True, metavar="FILE", help="Path to model file")
     parser.add_argument("-e", "--experiment", type=str, required=True, metavar="FILE", help="Experiment config file. From this the experiment class name is extracted.")
     parser.add_argument("-c", "--config", type=str, required=True, metavar="FILE", help="Path to yaml config training file")
+    parser.add_argument("-w", "--initial_weights", type=str, required=False, metavar="FILE", help="The path to the initial weights. These can be used by the model instead of the random initialization")
```

Comment:
```suggestion
    parser.add_argument("-w", "--initial_weights", type=str, required=False, nargs='?', const=None, default=None, metavar="FILE", help="The path to the initial weights. These can be used by the model instead of the random initialization")
```

so if the flag is present but nothing is beside it it is going to be defaulted to None.